binary_info/blink_any/blink_any.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+
+#ifdef CYW43_WL_GPIO_LED_PIN
+#include "pico/cyw43_arch.h"
+#endif
+
+// Set an LED_TYPE variable - 0 is default, 1 is connected to WIFI chip
+// Note that LED_TYPE == 1 is only supported when initially compiled for
+// a board with PICO_CYW43_SUPPORTED (eg pico_w), else the required
+// libraries won't be present
+#if defined(PICO_DEFAULT_LED_PIN)
+    bi_decl(bi_ptr_int32(0x1234, 0x5678, LED_TYPE, 0));
```

Comment:
I wonder if it's worth adding a note about what these numbers are for?

Reply:
I've changed the examples to actually use feature groups, which I think shows what those numbers are for? It still uses arbitrary ones (0x1111 and 0/1), but shows their use